### DISCUSSION TITLE
Can I create start point?

### DISCUSSION THREAD

**Discussion by pleshevskiy** [terrastruct/d2#286] [Q&A]
I'm trying to create a diagram for my project with d2 and I want to create a starting point, but I don't understand how to do it.

![image](https://user-images.githubusercontent.com/7839004/204899041-84a6f276-c8ab-49d4-848b-99965fe2f6e7.png)

**Comment by rickyteng** [comment]
Like this ?

classes: {
  init: {
    label: ""
    shape: circle
    style.filled: true
    style.fill: "#0D32B2"
    style.stroke-width: 0
    width: 15
  }
}

init.class: init
init -> Locked
Locked -> Locked: Push
Locked -> Unlocked: Coin
Unlocked -> Unlocked: Coin
Unlocked -> Locked: Push
